lib/request/GetRequestV2.js

```diff
@@ -190,7 +197,7 @@ GetRequestV2.prototype = {
         model._path = emptyArray;
 
         // flatten all the requested paths, adds them to the
-        var nextPaths = flattenRequestedPaths(requested);
+        var nextPaths = model._useServerPaths ? data.paths : flattenRequestedPaths(requested);
```

Comment:
if requested paths need flattening might data.paths need flattening too?

Reply:
It doesn't, only because it's really just removing an outer wrapped on requested that shouldn't be present on the envelope paths.